releases/latest/beta/bundle.yaml

```diff
@@ -10,7 +10,7 @@ applications:
     _github_repo_name: admission-webhook-operator
   argo-controller:
     charm: argo-controller
-    channel: latest/beta
+    channel: 3.3/stable
```

Comment:
I suggest we leave this as latest/beta. [This PR](https://github.com/canonical/argo-operators/pull/141) should fix the issue we observed during our testing phase, and the fix is now in both latest/edge and latest/beta.

Reply:
agreed. based on this, should we also update it in 1.8/beta?

Reply:
I'd say so, just so we don't forget.